.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ jobs:
             make docker.tag generate_yaml
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: PATH=$GOPATH/bin:$PATH make e2e_simple_run E2E_ARGS="--skip_cleanup -use_local_cluster -cluster_wide -alsologtostderr -test.v -v 2"
+      - run: PATH=$GOPATH/bin:$PATH make e2e_simple_run E2E_ARGS="--skip_cleanup -use_local_cluster -cluster_wide -test.v -v 2"
```

Comment:
is the leftover `-v 2` doing anything now ? will it break ?

Reply:
It wasn't clear that all the -v X flags were indeed for glog, so I submitted as-is. Clearly, they are indeed for glog. I'll update and try again.

Reply:
it might be safer to just absorb (or even better translate into level == debug ) the -v in case some of our customers have their own templates for installing istio ?

(@costinm thoughts on backward compatibility needs for this ?)

Reply:
ps: you missed this line's "-v 2" - and should those be replaced by -loglevel debug?